src/express/commands/milestone.js

```diff
@@ -0,0 +1,400 @@
+/* eslint-disable dot-notation */
+import { promises } from 'fs-extra'
+import { loadDevnetConfig, splitToArray } from '../common/config-utils'
+import { timer } from '../common/time-utils'
+import { checkLatestMilestone } from './monitor';
+
+const {
+  runScpCommand,
+  runSshCommand,
+  maxRetries,
+  runSshCommandWithoutExit,
+  runSshCommandWithReturn
+} = require('../common/remote-worker')
+
+const milestoneLength = 64
```

Comment:
check for this value

Reply:
For devnets in matic-cli it's still 64. I remember Vaibhav mentioning that it's reduced to 12 now, but the milestone which are proposed are still using 64.

Reply:
@VAIBHAVJINDAL3012 please check this

Reply:
Actually Manav forgot to take the pull, We discussed over it.